### ISSUE TITLE
Define criteria for success / failure in inbound

### ISSUE THREAD

**Issue by be-ant** [ST3AM-US/st3am-com-issues#20]
When updating activities in OFS via BulkUpdate there are three outputs:
- warnings: potential errors. Activities have been created but with a warning
- errors: Activities were not created
- success: Activities created without warnings

For an  inbound file containing N activities there are multiple potential outcomes, that should be identified as success or error:

- 0 errors (it is assumed SUCCESS)
- 0 errors, K activities with warnings, N -K activities successfully created with no errors (it is assumed SUCCESS)
- K activities in errors (K<N)
- N errors, no activities on warning or success

**Comment by DelphinoVarela** [comment]
Criteria :
- for N activities
- warnings can be N
- errors must be 0

**Comment by be-ant** [comment]
Assuming that you mean Success criteria is no errors.

What should be then the action when there are errors (> 1 and < N) ? At that moment some activities will have been created, so the original file contains activities that are OK and activities that haven't been created. Should we in this case delete the original file or leave it?

**Comment by DelphinoVarela** [comment]
Let's keep it simple : 
1- somehow we should easily be aware about the error(s),
2- the file should be archived to not block the all process and for later manual reprocessing.

Is it okay?

**Comment by be-ant** [comment]
Yes, the original file is always stored internally. So, for the partial (some errors, some good) then we will continue the process with the good ones AND create an .errors file. I will also check if we can send an email notifying about the errors.
